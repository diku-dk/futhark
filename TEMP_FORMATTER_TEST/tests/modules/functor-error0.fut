-- Error when passing a struct of the wrong signature to a functor.
-- ==
-- error: x.*i32.*f32

module F (P: { val x: i32 }) = {
  def x: i32 = P.x + 1
}

module F' = F({def x: f32 = 1.0f32})

def main(): i32 = F'.x
