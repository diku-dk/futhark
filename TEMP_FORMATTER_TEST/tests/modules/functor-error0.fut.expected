5e77964608468f2d294cbbcfed43859b
